tools/run_tdvt_tests.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+project_dir="$( cd "$(dirname "$0")/.." >/dev/null 2>&1 ; pwd -P )"
+readonly project_dir
+target_dir="$project_dir/target/"
```

Comment:
Is this directory predefined? If not, maybe give it  mare specific name?

Reply:
No. This is just a naming convention for the output / build directory.